cirq-core/cirq/sim/clifford/clifford_simulator_test.py

```diff
@@ -343,21 +343,26 @@ def test_clifford_circuit():
 def test_clifford_circuit_2(qubits):
     circuit = cirq.Circuit()
 
-    np.random.seed(2)
+    x_list = [0, 5, 0, 6, 3, 2, 3, 0, 2, 1, 3, 5, 2, 4, 4, 4, 5, 3, 6, 4, 2, 3, 3, 6, 2, 1, 2, 4, 3, 5, 0, 4, 6, 6, 3, 1, 2, 0, 4, 6, 4, 2, 4, 2, 1, 6, 0, 2, 5, 2]
+    if(len(qubits) == 4):
+        in_qubits = [0, 3, 1, 0, 2, 3, 2, 3, 0, 3, 2, 1, 3, 3, 1, 3, 3, 3, 2, 0, 0, 0, 1, 3, 3, 2, 0, 2, 3, 3, 3, 2, 2, 1, 2, 0, 3, 3, 1, 0, 0, 2, 2, 3, 3, 1, 3, 2, 0, 0]
+    else:
+        in_qubits = [0, 1, 1, 0, 0, 1, 0, 1, 0, 1, 0, 1, 1, 1, 1, 1, 1, 1, 0, 0, 0, 0, 1, 1, 1, 0, 0, 0, 1, 1, 1, 0, 0, 1, 0, 0, 1, 1, 1, 0, 0, 0, 0, 1, 1, 1, 1, 0, 0, 0]
```

Comment:
Using this really long unrolled list, doesn't exactly make this the most interpretable test. Could we possibly shorten up the number of operations ? Or maybe instead of modifying the global random seed we could introduce some fixed local random seed (https://numpy.org/doc/stable/reference/random/legacy.html?highlight=randomstate#numpy.random.RandomState is gauranteed legacy support) and indicate that is specifically chosen to produce a "well behaved" Clifford circuit ?

Reply:
Hi @MichaelBroughton. We used the suggested function, which should ensure that the simulator runs on the same circuit every time without setting the seed inside the simulator itself. Please let us know if there is anything else that should be done. Thank you.

Reply:
Hi @MichaelBroughton. I would also like to add that using the local random seed function (while ensuring a more stable test) will allow the function to test a complex circuit, rather than one with less components. Please let us know if there is anything else that should be done or if the PR is ready to be merged. Thank you.